guides/source/api_app.md

```diff
@@ -122,7 +122,7 @@ For example, the Rails boot process is what uses your `config/database.yml` file
 when configuring Active Record.
 
 **The short version is**: you may not have thought about which parts of Rails
-are still applicable even if you remove the view layer, but the answer turns out
+is still applicable even if you remove the view layer, but the answer turns out
```

Comment:
Can you revert this change? It was correct before because "parts" is plural and "Rails" is the proper noun, so "are" refers to "parts" not "Rails".